tests/conftest.py

```diff
@@ -437,7 +437,7 @@ def __init__(
         self.model = LLM(
             model=model_name,
             tokenizer=tokenizer_name,
-            trust_remote_code=True,
+            trust_remote_code="falcon" not in model_name,
```

Comment:
Why we have to re-download Falcon?

Reply:
`trust_remote_code` did not work for falcon, not sure why